Source/Extensions/Blazorise.Markdown/Markdown.razor.cs

```diff
@@ -315,8 +318,8 @@ public async Task UpdateFileEndedAsync( IFileEntry fileEntry, bool success, File
 #pragma warning disable CS4014 // We want to let execution complete but wait for TaskCompletionSource on the background.
             Task.Run( async () =>
```

Comment:
Do we still need this `Task.Run`?

Reply:
Maybe we don't, I'll check

Reply:
So, it doesn't work, because **UpdateFileEndedAsync** is part of the File Write Operation (which is called in **ImageUploadChanged**) and this is the big original problem. 
So **ImageUploadChanged** won't finish until **UpdateFileEndedAsync** has finished. So that's why we set up the background task, to make everything finish and we secretly wait for TCS which is right after **ImageUploadChanged** (**UpdateFileEndedAsync / ImageUploadEnded**) which means URL should already been set.

Reply:
It doesn't work by removing the **Rask.Run** I mean.

Reply:
Not a big fan of `Task.Run` but if there is no other way I guess we can leave it.

Reply:
Well if you got other ideas I'm all ears 😅

Reply:
I dont haha